### PR TITLE
toss speed that are absurd.  speed > 150kph or speed < 10kph.

### PR DESCRIPTION
**Pull request by gknisely**

### CODE REVIEW

File: edges.lua [hunk 1]
```diff
@@ -331,6 +331,10 @@ function normalize_speed(speed)
   --grab the number prefix
   local num = numeric_prefix(speed)
 
+  --if num > 150kph or num < 10kph....toss
```
Comment (kevinkreiser) [inline edges.lua]: should do this after the conversion from mph

Comment (gknisely) [inline edges.lua]: doh...i need coffee...hold

**Comment by gknisely** [comment]
@kevinkreiser check it

**Comment by kevinkreiser** [comment]
:rocket:

**Pull request merged**